examples/viewer/package.json

```
@@ -46,7 +47,7 @@
     "clean": "rm -rf dist/",
     "start": "webpack-dev-server --config ./config/webpack.dev.js",
     "test": "echo 'There are no tests configured for this project!' && exit 0",
-    "build": "webpack --config ./config/webpack.prod.js",
+    "build": "webpack --config ./config/webpack.dev.js",
```

you can use `npm run build:dev` for development config
```suggestion
    "build": "webpack --config ./config/webpack.prod.js",
```